.github/workflows/publish-documentation-new-version.yml

```diff
@@ -7,6 +7,8 @@ jobs:
     name: Publish Documentation with New Version
     runs-on: ubuntu-latest
     steps:
+      - name: Set env.BRANCH
+        run: echo "BRANCH=$(echo $GITHUB_REF | cut -d'/' -f 3)" >> $GITHUB_ENV
```

Comment:
Why did you add this step ?

Reply:
I did not add it. I just did a rebase in origin/main-vnext. I suppose that it was added by kevin in main-vnnext branch

Reply:
I dropped the commit ok kevin on your branch